scripts/mkcloud

```diff
@@ -889,7 +887,7 @@ function setup_ironic_test_env
         # create node in ironic
         onadmin create_ironic_node $cloud-node$i \
             $vcpus $ram_mb $ironicnode_hdd_size $(macfunc $i 1) \
-            $deploy_interface \
+            auto \
```

Comment:
This is the only place where this parameter is passed, so why keep it at all? May as well just always make oncontroller_create_ironic_node automatically select the interface.

Reply:
Hmmm... we may... I think I just don't like "shrinking" interfaces :o)